src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -553,6 +553,13 @@ ov::SupportedOpsMap Plugin::query_model(const std::shared_ptr<const ov::Model>&
 
 std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& model_stream,
                                                          const ov::AnyMap& config) const {
+    return import_model(model_stream, nullptr, config);
```

Comment:
Which version is used with null pointer of buffer or context?

Reply:
When AlignedBuffer with mmap is nullptr, legacy implementation with istream only is used